.coderabbit.yaml

```
@@ -52,20 +54,41 @@ path_filters:
 # Knowledge base settings
 knowledge_base:
   # Opt in to learnings from your codebase
-  learnings: true
+  learnings:
+    enabled: true
   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)